news/2025-02-13-conda-forge-v1-recipe-support.md

```diff
@@ -0,0 +1,84 @@
+# Announcing the new recipe format on conda-forge
+
+The conda-forge team is excited to announce that the v1 recipe format is available on conda-forge. The v1 recipe format is a community initiative dating back over 3 years to improve the recipe format for conda packages. If you are a maintainer of a feedstock on conda-forge, you have probably dealt with `meta.yaml` files that conda-build utilizes. The file format has some limitations which is why the community has come together to come up with an improved version of the format: the v1 format.
+
+## rattler-build and the v1 spec
+
+The v1 recipe format has a number of benefits:
+
+- It always parses as valid YAML which makes it much easier to modify it with the bot infrastructure of conda-forge. While meta.yaml looks like YAML, it can contain Jinja logic incompatible with the YAML specification, which make significantly complicates parsing and automated manipulation. Additionally, the recipe has a published JSON schema which means that the editing experience in VSCode is greatly improved with contextual help.
+- The new recipe executes much faster, which is partially due to some design decisions (no more recursive evaluation) as well as due to rattler-build building on top of a modern re-implementation of the conda standards in Rust.
+- The log output of rattler-build is greatly improved, always showing the user what the final files in the package are and the final metadata.
+- Some features of conda-build, such as multiple outputs, had a lot of implicit behavior. We are fixing that in the v1 recipe.
```

Comment:
These three bullet points need to separate out two distinct things, namely the format versus the tools that use the format.

Reply:
Not sure ... this blog post is about the integration in conda-forge and that relies on rattler-build a lot. This is not only about the v0 vs v1 format ...